go/cloudbuild.yaml

```diff
@@ -19,7 +19,7 @@ steps:
 # Go 1.11 build
 - name: gcr.io/cloud-builders/docker
   args: ['build', '-t', 'gcr.io/$PROJECT_ID/go111', '.']
-  dir: go/go111
+  dir: go111
```

Comment:
I suppose this is all relative to where this `gcloud builds submit` is run from? I _think_ automation will run after PR is submitted to upload these images. Maybe this is not hooked up for go since we recently moved our images here?

Reply:
I don't believe there is any CI of any sort set up against this repository, testing, or release 😰

Reply:
If that is the case, we should get the readme updated: 

> Once the PR is merged, Cloud Build will rebuild the images for any images that were updated. See the project cloud-devrel-kokoro-resources to edit or add Cloud Build triggers.

Reply:
Seems we do have builds set up? Either way, I reverted the change.